### PR TITLE
TAL ingestion

### PR DESCRIPTION
**Pull request by xinru1414**

### CODE REVIEW

File: data/yaml/venues/tal.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+acronym: Tal
```
Comment (mjpost) [inline data/yaml/venues/tal.yaml:1]: Should be TAL.

**Comment by mjpost** [comment]
Please see the comments on #2339. We need to mark this as a journal so that the BibTeX gets generated correctly.

**Comment by mcandito** [comment]
Hi @xinru1414, @mjpost,

I've studied the preview of the ingestion

I've seen a **bug** on my side (mismatch of pdfs for the varia issues)

Could you please consider these two **debugged** archives:
http://www.linguist.univ-paris-diderot.fr/~mcandito/tmp/2011.zip
http://www.linguist.univ-paris-diderot.fr/~mcandito/tmp/2018.zip

 (I've changed Tal to TAL in the meta files)

Moreover, I am confused about the procedure:
in the zip files I produce, I create the bibtex files.
But it seems they are not taken into account.
Are you supposed to produce them??

Thank you

**Comment by mbollmann** [comment]
@mcandito We generate the BibTeX from the XML files to ensure consistency between them; we do not consider external BibTeX files.

**Comment by mcandito** [comment]
> @mcandito We generate the BibTeX from the XML files to ensure consistency between them; we do not consider external BibTeX files.

Hi @mbollmann,
I provide archives in ACLPUB format (cf. https://acl-org.github.io/ACLPUB/anthology.html)
which contain no XML file, but some pdfs plus bibtex files.

Do you mean you use the ACLPUB format to recreate an XML file, and then recreate the bibtex files?
Thanks

**Comment by mbollmann** [comment]
Yes, that's what I meant. The purpose of the ACLPUB tool is to create our XML files, like the ones you can see here in this pull request: https://github.com/acl-org/acl-anthology/pull/2384/files#diff-55ff300068d107eba99b1482550116a290a7241f39e6b08b148553c067d81590 

The BibTeX you see* on the website is generated from that.

(*or rather, should see, as Matt's recent comments indicate it's not working correctly yet)

File: data/xml/2011.tal.xml [hunk 2]
```diff
@@ -0,0 +1,260 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2011.tal">
+  <volume id="1" ingest-date="2023-02-10">
+    <meta>
+      <booktitle>Traitement Automatique des Langues 2011 Volume 52 Numéro 1</booktitle>
+      <editor><first>Éric Villemont</first><last>de La Clergerie</last></editor>
```
Comment (mcandito) [inline data/xml/2011.tal.xml:6]: Should be 
<editor><first>Éric</first><last>Villemonte de La Clergerie</last></editor>

Comment (mcandito) [inline data/xml/2011.tal.xml:6]: Should be
\<editor>\<first>Éric\</first>\<last>Villemonte de La Clergerie</last></editor>

**Comment by mcandito** [comment]
Hi @xinru1414, 

I've studied the preview of the ingestion

I've seen a bug on my side (mismatch of pdfs for the varia issues)

Could you please consider these two debugged archives and redo the ingestion test??
http://www.linguist.univ-paris-diderot.fr/~mcandito/tmp/2011.zip
http://www.linguist.univ-paris-diderot.fr/~mcandito/tmp/2018.zip

Then I am lost about the procedure, 
who is performing the "approving review" ?
am I supposed to do anything?

Thanks

File: data/xml/2011.tal.xml (hunk 2 above)
Comment (mjpost) [reply data/xml/2011.tal.xml:6]: ping @xinru1414

**Comment by mjpost** [comment]
Hi @mcandito—I'll approve this when it's ready. Sorry for the delay; we realized we'll need some coding to properly support generating journal BibTeX for these proceedings.

**Comment by mjpost** [comment]
Hi again, I found a little bit of time and this came together faster than I expected. @mcandito, can you verify that the looks like the correct generated BibTeX?

```bibtex
@article{lavallee-langlais-2011-moranapho,
    title = "Moranapho: un syst{\`e}me multilingue d{'}analyse morphologique bas{\'e} sur l{'}analogie formelle [Moranapho: a multilingual system for morphological analysis based on formal analogy]",
    author = "Lavall{\'e}e, Jean-Fran{\c{c}}ois  and
      Langlais, Philippe",
    journal = "Traitement Automatique des Langues",
    volume = "52",
    number = "2",
    year = "2011",
    address = "France",
    publisher = "ATALA (Association pour le Traitement Automatique des Langues)",
    url = "https://aclanthology.org/2011.tal-2.2",
    pages = "17--44",
    language = "French",
}
```

**Comment by mcandito** [comment]
> Hi again, I found a little bit of time and this came together faster than I expected. @mcandito, can you verify that the looks like the correct generated BibTeX?
> 
Hi @mjpost , yes the BibTeX is correct!

**Comment by mcandito** [comment]
> Hi @mcandito—I'll approve this when it's ready. Sorry for the delay; we realized we'll need some coding to properly support generating journal BibTeX for these proceedings.

Thanks @mjpost, I did not mean to hurry you ! Now I understand the procedure better.
Please note that I updated the zips on my side, to correct a mismatch between pdfs and article ids, for the varia issues.
Before approving, could you consider reingest all over again?? ping  @xinru1414 
http://www.linguist.univ-paris-diderot.fr/~mcandito/tmp/2011.zip
http://www.linguist.univ-paris-diderot.fr/~mcandito/tmp/2018.zip
Thanks

**Comment by mjpost** [comment]
I think so. It seems 2011 PDFs are off.

**Comment by mjpost** [comment]
2018 seems fine, though. @mcandito can you clarify?

**Comment by mcandito** [comment]
@xinru1414 @mjpost 
I confirm both are to reingest (actually the mismatch bug concerns 2011.tal-1 and 2018.tal-1)
Thanks

**Comment by mjpost** [comment]
@xinru1414 did you push PDFs? There are still wrong ones e.g., here: https://preview.aclanthology.org/tal-ingestion/volumes/2011.tal-3/. Please go through each volume and check 3–4 of them and then ping me.

**Comment by mjpost** [comment]
@mcandito the PDFs are off here, too, can you fix this? https://preview.aclanthology.org/tal-ingestion/volumes/2011.tal-3/

**Comment by mcandito** [comment]
@mjpost @xinru1414 I've fixed the new pdf mismatch for 2011.tal-3, sorry.
I've updated the data here: http://www.linguist.univ-paris-diderot.fr/~mcandito/tmp/2011.zip

**Comment by mjpost** [comment]
@xinru1414 is this then ready?

**Review by mjpost — APPROVED** [review]

**Pull request merged**